tests/ballerina-tools-integration-test/src/test/java/org/ballerinalang/test/run/BalxRunFunctionPositiveTestCase.java

```diff
@@ -95,10 +98,28 @@ public void testMultipleParam() throws BallerinaTestException {
         outLogLeecher.waitForText(2000);
     }
 
+    @Test
+    public void testArgWithColons() throws BallerinaTestException {
+        String arg = "test arg";
+        String fileName = "test_entry_function_with_colons.bal";
+        String[] args = {"-o",
+                tempProjectDirTwo.toString().concat(File.separator).concat("test_entry_function_with_colons"),
```

Comment:
Shall we improve the formatting in this line?

Reply:
Any suggestions? The following are even less readable IMO:
```java
        String[] args = {"-o", tempProjectDirTwo.toString().concat(File.separator)
                .concat("test_entry_function_with_colons"), (new File("src/test/resources/run/balx/complex"))
                .getAbsolutePath().concat(File.separator).concat(fileName)};
```

```java
        String[] args = {"-o", tempProjectDirTwo.toString().concat(File.separator)
                .concat("test_entry_function_with_colons"), 
                (new File("src/test/resources/run/balx/complex"))
                        .getAbsolutePath().concat(File.separator).concat(fileName)};
```

Reply:
IMO, existing one is better :)

Reply:
Cant we use something like -

```java
Paths.get(tempProjectDirTwo.toString(),"test_entry_function_with_colons"), new File("src/test" + 
 "/resources/run/balx/complex").getAbsolutePath(),fileName).toString();
```

Then you dont have to concat the `File.separator` as well.

Reply:
@MaryamZi `File.separator` will cause issues in windows environments. 
+1 for @Shan1024's suggestion for using `Paths.get()`.

Applicable for other places places as well..